utils/save_segments_in_flac.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2021  SpeechColab Authors
+
+import argparse
+from pathlib import Path
+from importlib.util import find_spec
+
+from speechcolab.datasets.gigaspeech import GigaSpeech
+
+if find_spec('audioread') is not None:
+    import audioread
+else:
+    raise ImportError('Need optional dependency: pip install audioread')
+
+if find_spec('soundfile') is not None:
+    import numpy as np
+    import soundfile as sf
+else:
+    raise ImportError('Need optional dependency: pip install soundfile')
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description='Save the audio segments into flac files.')
+    parser.add_argument('corpus', help='The corpus directory')
+    parser.add_argument('output', help='The output directory')
+    parser.add_argument('--subset', default='{XL}', help='The subset name')
```

Comment:
It would be great if we can make this argument type as "choice", and this tool may be particularly useful for extracting evaluation sets for benchmarking-only users.

Reply:
Yes, it's a good idea. I think we can let `speechcolab.datasets.gigaspeech` of PySpeechColab support multiple subsets. In the current version of PySpeechColab, only a single subset is allowed to be set.